docs/source/changelog.rst

```diff
@@ -81,29 +81,53 @@ Here is an enumeration of changes made since the last release and included in 6.
 - Fix for the terminal shutdown issue (:ghpull:`4180`)
 - Add japanese translation files (:ghpull:`4490`)
 - Workaround for socket permission errors on Cygwin (:ghpull:`4584`)
+- Implement optional markdown header and footer files (:ghpull:`4043`)
+- Remove double link when using custom_display_url (:ghpull:`5544`)
+- Respect cell.is_editable during find-and-replace (:ghpull:`5545`)
```

Comment:
I think we need some formatting wrappers for some of these, e.g. ` ``cell.is_editable`` `